17 primos/listAllPrime.java

```diff
@@ -28,12 +28,12 @@
       }
     }
   }
-
-}
-class ListAllPrime{
+ 
   public static void main(String args[]){
     int m;
     m=Integer.parseInt(args[0]);
-    Prime p=new Prime(m);
+    ListAllPrime.prime(m);
```

Comment:
prime is static, so you could call only using the method name (prime), because main is static to.